rest-api/test/unit_test/api_test/participant_api_test.py

```diff
@@ -8,6 +8,7 @@ class ParticipantApiTest(FlaskTestBase):
   def setUp(self):
     super(ParticipantApiTest, self).setUp()
     provider_link = {
+      "awardeeId": 2,
```

Comment:
provider link shouldn't have an awardeeId field on it... that's supposed to go on participant or participant_summary.